src/components/Page.js

```diff
@@ -23,11 +23,17 @@ const FloatFooterPageComponentWrapper = styled('div')`
   display: flex;
 `;
 
-const Page = ({ Head = Header, Foot = Footer, Component, ...props }) => (
+const Page = ({
+  Head = Header,
+  Foot = Footer,
+  Component,
+  cssFloatFooterPageComponentWrapper = ``,
+  ...props
+}) => (
   <Container>
     <Head />
     <FloatFooterPageContentWrapper>
-      <FloatFooterPageComponentWrapper>
+      <FloatFooterPageComponentWrapper css={cssFloatFooterPageComponentWrapper}>
```

Comment:
This will clash with the refactor I'm doing since I'm removing the possibility of injecting css using the `css` property.

Can you use a `className` with standard css (or less, sass, etc...) or a `style` instead?

Reply:
for now, to avoid clash and all, I won't style the spinner. I'll wait untill you finish your refactor

Reply:
I will avoid styling the spinner for now.